Fix build with FFmpeg 2.0.

Upstream status: Better patch sent and merged upstream.

Index: mpd-0.17.4/src/decoder/ffmpeg_decoder_plugin.c
===================================================================
--- mpd-0.17.4.orig/src/decoder/ffmpeg_decoder_plugin.c
+++ mpd-0.17.4/src/decoder/ffmpeg_decoder_plugin.c
@@ -47,6 +47,10 @@
 #undef G_LOG_DOMAIN
 #define G_LOG_DOMAIN "ffmpeg"
 
+#ifndef AVCODEC_MAX_AUDIO_FRAME_SIZE
+#define AVCODEC_MAX_AUDIO_FRAME_SIZE 192000 // 1 second of 48khz 32bit audio
+#endif
+
 static GLogLevelFlags
 level_ffmpeg_to_glib(int level)
 {
